### DISCUSSION TITLE
Imported score has 0 scores and an "Unknown mod (AA)" applied

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#17688] [Q&A] [answered]
The score in question: 
https://mega.nz/file/WZAVXBzT#mp03Fel906nZtH78_yQfcccNVBs1BvW3E2vCJo-qNDw (Exported from stable)
https://mega.nz/file/LIxA2Tqb#sR3HoT8aklzslGLWRkRwxANogWFmogFeHRsrqm1yEfQ (Exported from lazer)

I pressed "Import scores from stable" and happened to find this score for [this](https://osu.ppy.sh/beatmapsets/1322944#osu/2988681) map in the local leaderboards in osu!(lazer). (The cursor doesn't show up for some reason, I don't know if it's intentional. The screenshots were taken with `F12`)

![osu_2022-04-06_23-40-41](https://user-images.githubusercontent.com/11213135/162026737-caab0bc5-45bb-46e6-8129-1d8166d47d9d.png)
![osu_2022-04-06_23-40-47](https://user-images.githubusercontent.com/11213135/162026752-7326ef79-9b15-49b0-a8ce-d3f9850d2d98.png)
![osu_2022-04-06_23-40-32](https://user-images.githubusercontent.com/11213135/162028907-6b2db2f3-b3ec-42a4-b7a7-d05f9a213ab9.png)

I first tried to export the score in lazer into stable, and here's the result screen of the score

![screenshot009](https://user-images.githubusercontent.com/11213135/162028276-24a384a4-2c93-4bc3-a5fc-7291f9fe4e63.png)

and its leaderboards score in the local leaderboards

![screenshot010](https://user-images.githubusercontent.com/11213135/162028364-9fe4cdfb-7f2d-4ea4-83be-b2c70b1243b5.png)

So next I tried to watch the replay in lazer, and this is what happened

https://user-images.githubusercontent.com/11213135/162029493-662f425d-7223-42f9-a6ac-608bee42020b.mp4

But the weirdest thing out of all of this is how the replay "behaved" in stable. (File is bigger than 10MB so I apologize for the third-party link)

https://streamable.com/fzxo26

The cursor seems to be misaligned from the center for some reason and doesn't fail until the end even without No Fail on.

I don't know if this is a known issue with a reproducible procedure, but it's an interesting case worth documenting in my opinion.

**Comment by bdach** [comment, marked as answer]
This score did not come from stable.Iit was a score set on lazer with the "aim assist" mod on, which was later renamed to "magnetised" in https://github.com/ppy/osu/pull/17588. Because the mod acronym changed, old replays that used said mod cannot be played back anymore.

This also explains why the cursor is "misaligned". It is simply because that's what the mod did (moved objects closer to mouse).

**Reply by ikkue** [reply]
That would be the perfect explanation! So could there be a potential fix in the future for this? Like somehow forcing all the plays with "Aim Assist" to be assigned with "Magnetised" instead?

P.S. This could also be expanded upon to "how should replays with newer mods from osu!(lazer) be displayed in osu!(stable) which doesn't have said mods", but I presume that should be a different discussion.

**Reply by bdach** [reply]
Not sure. We could write a migration to fix it in post but honestly not sure it's worth the effort. @peppy thoughts?

**Reply by smoogipoo** [reply]
Not worth fixing.
